CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## HEAD
 
+- @cosmjs/cli: Now supports HTTPs URLs for `--init` code sources.
```

Comment:
This is a great feature add.

Do you want to test that one time in the CI? Or would it be too hard to find some stable code to test against? 

I mean, I guess generate_address should be relatively stable and we could point to master until some PR needs to point to it's own branch?

Reply:
Yeah, testing against an external resource is a problem. I'd leave it as is and use a local HTTP server is we really have problem with this.